JoinColony/colonyCDapp
# Fix: Show new colony details on successful colony details update

## Description

![Update avatar](https://github.com/user-attachments/assets/b9e972fe-d0d7-4890-9c7c-68cf0cd4b259)

## Testing

1. Open the Colony Switcher and take note of the your Colony's avatar
2. Bring up the Action Form and set it to Edit Colony Details
3. Upload a different Colony avatar
4. Fill in all other required form fields and submit the form
5. After the Action is successfully completed, open the Colony Switcher
6. Verify that the Colony Avatar is updated

Resolves #3096
